data/human/intro missions.txt

```diff
@@ -508,3 +508,95 @@ conversation "end of intro missions"
 		
 	label end
 	`	He hands you a credit chip worth <payment>, and says, "Best of luck to you, Captain. Don't hesitate to look me up if you're ever in the area. And remember to check the spaceports wherever you land. You never know who you'll find wandering around there."`
+
+
+
+mission "Intro [Mining]"
+	landing
+	name "Explore Mining"
+	description "Learn about mining."
+	source "Sunracer"
+	to offer
+		or
+			has "ships: Interceptor"
+			has "ships: Drone"
+			has "ships: Fighter"
+			has "ships: Light Warship"
+			has "ships: Medium Warship"
+			has "ships: Heavy Warship"
+			has "ships: Superheavy"
+	
+	on offer
+		conversation
+			`	On your approach to <planet> you notice a large warehouse with private landing pads for the Clementine Metals Company. Someone there would know about mining asteroids. Would you like to ask them?`
```

Comment:
```suggestion
			`	As you approach <planet>, you spot a large warehouse belonging to the Clementine Metals Company. Maybe someone there could give you some tips on mining. Do you want to look around?`
```
I'm not sure how to say the private landing pad thing.

Reply:
Restricted Berths should be a good substitute.

Reply:
My problem isn't with the phrasing; it's with how you can tell their restricted without being awkward.

Reply:
ATC telling you it's restricted airspace would be good.